crates/ra_ide_api/Cargo.toml

```diff
@@ -20,6 +20,7 @@ jemallocator = { version = "0.1.9", optional = true }
 jemalloc-ctl = { version = "0.2.0", optional = true }
 
 ra_syntax = { path = "../ra_syntax" }
+ra_parser = { path = "../ra_parser" }
```

Comment:
The macro should be re-exported in ra_syntax, such that no direct dependency on `ra_parser` is required.

Reply:
I didn't figure out how to do this last time. It's correct right now